Tasks/GitHubReleaseV0/operations/ChangeLog.ts

```diff
@@ -15,7 +15,7 @@ export class ChangeLog {
      * @param compareWithRelease
      * @param changeLogCompareToReleaseTag
      */
-    public async getChangeLog(githubEndpointToken: string, repositoryName: string, target: string, top: number, compareWithRelease: ChangeLogStartCommit, changeLogCompareToReleaseTag?: string): Promise<string> {
+    public async getChangeLog(githubEndpointToken: string, repositoryName: string, target: string, top: number, compareWithRelease: ChangeLogStartCommit, changeLogCompareToReleaseTag?: string, changeLogLabels?: any): Promise<string> {
```

Comment:
why type is any?

Reply:
User input is a string which is then JSON parsed.